Story/Episode 1/st_010180.csv

```diff
@@ -5,26 +5,26 @@ st_010180_0020#0,"""Internaron a esa chica en el centro médico, así que deber
 st_010180_0030#0,"""Me pregunto cual sera el resultado de nuestro examen..."""
 st_010180_0040#0,"""Supongo que no debería preocuparme por eso.<br>Probablemente saldrá bien."""
 st_010180_0050#0,"""De todos modos, voy a estar en el vestíbulo por un tiempo. Nos vemos más tarde!"""
-st_010180_0060#0,"""Tu eres <%me>?"""
+st_010180_0060#0,"""Hola, ¿puedes escucharme?, ¿Tu eres <%me>?"""
 st_010180_0070#0,"""Soy parte del personal médico. Me llamo Philia."""
 st_010180_0080#0,"""La chica que encontraste en Naberius acaba de despertar hace un momento."""
 st_010180_0090#0,"""¿Podrías venir al el centro médico y visitarla?"""
-st_010180_0100#0,"""<%me>, ¿Verdad? Gracias por venir."""
-st_010180_0110#0,"""La chica ... no ha hablado ni una palabra desde que se desperto."""
+st_010180_0100#0,"""<%me>, ¿Verdad? gracias por venir."""
+st_010180_0110#0,"""La chica... no ha hablado ni una palabra desde que se desperto."""
 st_010180_0120#0,"""…………"""
 st_010180_0130#0,"""……<%me>。"""
 st_010180_0140#0,"""¿Le dijiste tu nombre?"""
 st_010180_0150#0,"""...Lo escuché... Dentro de mi cabeza."""
 st_010180_0160#0,"""...Mi nombre es... Matoi."""
 st_010180_0170#0,"""No hay coincidencias en la base de datos. Al menos no en la base de datos ARKS..."""
 st_010180_0180#0,"""¿Podría ser de otro planeta? a primera vista, pensé que ella era un miembro de ARKS."""
-st_010180_0190#0,"""Matoi, ¿de donde eres? ¿En qué planeta vives?"""
-st_010180_0200#0,"""……う…………<br>……<%me>"""
+st_010180_0190#0,"""Matoi, ¿De donde eres? ¿En qué planeta vives?"""
+st_010180_0200#0,"""……う... ...<br>……<%me>"""
```

Comment:
Aahh, do you want CJK in your patch?

Reply:
Generally, they do not bother me, but in that specific case, it seemed to me that they did not look good.